examples/wflow_build.ini

```diff
@@ -13,9 +13,17 @@ basin_index_fn  = merit_hydro_index # source of basin index corresponding to hyd
 upscale_method  = ihu           # upscaling method for flow direction data, by default 'ihu'
 
 [setup_rivers]
-hydrography_fn  = merit_hydro   # source hydrography data, should match basemaps source
-river_upa       = 30            # minimum upstream area threshold for the river map [km2]
-slope_len       = 2000          # length over which tp calculate river slope [m]
+hydrography_fn   = merit_hydro      # source hydrography data, should correspond to hydrography_fn in setup_basemaps
+river_geom_fn    = rivers_lin2019_v1 # river source data with river width and bankfull discharge
+river_upa        = 30               # minimum upstream area threshold for the river map [km2]
+rivdph_method    = powlaw           # method to estimate depth {'powlaw', 'manning', 'gvf'}
+min_rivdph       = 1                # minimum river depth [m]
+min_rivwth       = 30               # minimum river width [m]
+slope_len        = 2000             # length over which tp calculate river slope [m]
+smooth_len       = 5000             # length over which to smooth river depth and river width [m]
+
+[setup_floodplains]             # adds hydrologically conditioned elevation FloodplainZ
```

Comment:
@verseve Do you have a suggestion for a better name instead of `FloodplanZ`? What do you think of the following (also curions to hear your thoughts on this naming @laurenebouaziz @hboisgon):

- `wflow_dem` -> `dem_avg`
- `RiverZ` -> `dem_subgrid` This dem is based on the subgrid outlet elevation levels, hydrologically conditioned to D8 flowdirs and not just for the river cells anymore.
- `FloodplainZ` -> `dem_avg_d4` (or `dem_subgrid_d4`) depending on which of the above it s based on. This map has D4 conditioned elevation for land and D8 conditioned for river cells.

It depends on the `elevnt_map`  argument if the FloodplainZ is based on the subgrid or average dem, see
```
      Parameters
      ----------
      elevtn_map: str, optional
          Name of staticmap to base the river+floodplain elevation map on, by default
          "wflow_dem". To base it on the subgrid river elevation map use "RiverZ";
          to base it on avarage cell elevation use "wflow_dem".
```

Reply:
Thanks for the answers Dirk. I agree with your suggestions, with these names it is much easier to interpret these variables.

For the `setup_floodplain` my suggestion is to use for example `setup_hydrodem`.

Reply:
I agree that the suggested names are clear like this

Reply:
- I have decided to keep the `wflow_dem` name as it seems to be used quite often in other scripts (e.g. the example notebooks) and might require a larger discussion on naming before changing.
- The `RiverZ` map has been deprecated in favor of `dem_subgrid` map which is derived when upscaling the flow directions and not hydrologically conditioned. 
- All the hydrological conditioning happens in `setup_hydrodem` where the user can select which elevation to apply the conditioning to. If based on `wflow_dem` the name becomes `hydrodem_avg`; if based on `dem_subgrid` > `hydrodem_subgrid`.
- I've added much explanation about the different option in the `setup_hydrodem` docstring